src/api/spec/features/webui/requests_spec.rb

```diff
@@ -163,7 +163,16 @@
 
   context 'review' do
     describe 'for user' do
-      skip
+      let(:reviewer) { create(:confirmed_user) }
```

Comment:
It's already using sequence and nothing happens on the backend as it's not running :-)